aten/src/ATen/functorch/BatchRulesBinaryOps.cpp

```diff
@@ -200,11 +200,15 @@ static std::tuple<Tensor,optional<int64_t>> masked_select_batch_rule(
   return std::make_tuple(result, 0);
 }
 
-static std::tuple<Tensor,optional<int64_t>> masked_select_backward_batch_rule(
-    const Tensor& grad, optional<int64_t> grad_bdim,
-    const Tensor& self, optional<int64_t> self_bdim,
-    const Tensor& mask, optional<int64_t> mask_bdim) {
-  TORCH_CHECK(!mask_bdim.has_value(),
+static std::tuple<Tensor, optional<int64_t>> masked_select_backward_batch_rule(
```

Comment:
This is a just a formatting change. I think the change to remove this function and add a corresponding decomposition was missed in the commit.

Reply:
Does `lintrunner` applies to C/C++ files as well?

Reply:
Yes, IIRC correctly it runs clang-tidy on files under `aten`.

Reply:
Thanks. I guess this formatting change was done by my vscode setup. Will undo this change

Reply:
We should update the PR title to remove `masked_select_backward` and `onehot` as those are not handled in this PR.

Reply:
Done